eva-accession-release-automation/publish_release_to_ftp/publish_release_files_to_ftp.py

```diff
@@ -198,9 +201,10 @@ def publish_assembly_release_files_to_ftp(current_release_assembly_info, release
             )
         )
 
-    # Create a link from assembly folder to species_folder ex: by_assembly/GCA_000005005.5/zea_mays to by_species/zea_mays/GCA_000005005.5
-    create_symlink_to_species_folder_from_assembly_folder(current_release_assembly_info, release_properties,
-                                                          public_release_assembly_folder)
+    if current_release_assembly_info["num_rs_to_release"] > 0:
```

Comment:
Debatable whether this is even a good idea, but elsewhere we check both `num_rs_to_release` and `should_be_released`... should we do the same here to be consistent?

Reply:
I think you're right that this is inconsistent with what is done elsewhere.
The reason I've only used the `num_rs_to_release` is because originally the `should_be_released` flag really meant that the release pipeline should be run not that the data should be copied and in previous version of the release automation `should_be_released=false` and  `num_rs_to_release>0` was used to denote release that had not changed since the previous release and could be hard-linked.